frontend/public/src/containers/pages/CatalogPage_test.js

```diff
@@ -223,14 +223,14 @@ describe("CatalogPage", function() {
           },
           departments: [
             {
-              name:     "History",
-              courses:  0,
-              programs: 5
+              name:        "History",
+              course_ids:  [],
+              program_ids: [2, 2, 2, 2, 2]
```

Comment:
nit: should the program be different for each? technically this is 1 program, not 5.